draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -130,11 +140,138 @@ This document uses "service" and "workload" interchangeably. Otherwise, all term
 
 # Application Level Service To Service Authentication {#app-level}
 
-## The WIMSE ID Token
+## The Workload Identity Token {#to-wit}
+
+The Workload Identity Token (WIT) is a JWS {{RFC7515}} signed JWT {{RFC7519}} that represents the identity of a workload.
+It is issued by the Identity Server and binds a public key to the workload identity.
+A WIT MUST contain the following:
+
+* in the JOSE header:
+    * `alg`: An identifier for a JWS asymmetric digital signature algorithm
+     (registered algorithm identifiers are listed in the IANA JOSE Algorithms registry {{IANA.JOSE.ALGS}}). The value `none` MUST NOT be used.
+    * `typ`: the WIT is explicitly typed, as recommended in {{Section 3.11 of RFC8725}}, using the `wimse-id+jwt` media type.
+* in the JWT claims:
+    * `iss`: The issuer of the token, which is the Identity Server, represented by a URI.
+    * `sub`: The subject of the token, which is the identity of the workload, represented by a URI.
+    * `exp`: The expiration time of the token (as defined in {{Section 4.1.4 of RFC7519}}).
+      WITs should be refreshed regularly, e.g. on the order of hours.
+    * `jti`: A unique identifier for the token.
+    * `cnf`: A confirmation claim containing the public key of the workload using the `jwk` member as defined in {{Section 3.2 of RFC7800}}.
+     The WIT MUST NOT be used as a bearer token. Instead, it should be used in conjunction with one of the methods in {{dpop-esque-auth}} or {{http-sig-auth}}.
```

Comment:
```suggestion
The WIT MUST NOT be used as a single bearer token.
```
Otherwise it sounds like it should not be put into the Authorization header at all.

Reply:
Discussed offline.

Reply:
```suggestion
WITs should always be used in conjunction with {{dpop-esque-auth}} or {{http-sig-auth}}. For this reason it MUST NOT be used as a bearer token and not go into the `Authorization` header.
```

Reply:
thanks @arndt-s - using some of your wording there and some of my old wording mixed together to hopefully not be too terrible. But we can revisit later, if need be. 
```suggestion
     The workload MUST prove possession of the corresponding private key when presenting the WIT to another party, which can be accomplished by using it in conjunction with one of the methods in {{dpop-esque-auth}} or {{http-sig-auth}}. As such, it MUST NOT be used as a bearer token and is not intended for use in the `Authorization` header.
```